test/deploy/rackhd_ha_resource_install.py

```diff
@@ -0,0 +1,112 @@
+"""
+Copyright 2017, Dell EMC, Inc.
+This script prepares the RackHD HA cluster resources
+
+This script perform the following functions:
+    - configure mongo resource
+    - configure mongo virtual ips
+    - colocate mongo resource with virtual ip
+    - create mongo replica set
+
+usage:
+    python run_tests.py -stack <stack ID> -test deploy/rackhd_ha_resource_install.py -numvms <num>
+"""
+from jinja2 import Environment, FileSystemLoader
+import os
+import unittest
+import fit_path  # NOQA: unused import
+import fit_common
+
+nodelist = []   # list of active nodes in cluster
+numvms = int(fit_common.fitargs()['numvms'])
+err = []
+numrs = numvms - 1   # number of mongo resource
+vip_dict = {'mongo': [], 'rabbit': [], 'rackhd': []}
+
+
+class rackhd_ha_resource_install(unittest.TestCase):
```

Comment:
please add the line "   longMessage = True" here....

Doing so means that self.AssertXXX()'s will also dump the values of the compares and such. Doing this beats going in and changing all those AssertXXXs to include the compare-values in the message!